sc2/wasm/pre.js

```diff
@@ -0,0 +1,26 @@
+Module.preRun ||= [];
+
+Module.preRun.push(function () {
+    window.wasm_syncfs = () => {
+        FS.syncfs( /*populate=*/ false, err => {
+            if (err) {
+                alert("Saving to IndexedDB failed, saved game & preferences will not be persistent.");
```

Comment:
Just in case `err` contains a helpful error message:

```suggestion
                alert("Saving to IndexedDB failed, saved game & preferences will not be persistent.\n" + err);
```